src/main/java/game/Game.java

```diff
@@ -254,10 +261,12 @@ private void updateStartScreen(final List<String> input) {
 	private void updateGame(final List<String> input) {
 		if (input.contains("R") && System.currentTimeMillis() 
 				- restartTime > MINIMAL_RESTART_TIME) {
-			startGame();
```

Comment:
Yes, the game should be restarted after the start screen. I found it in the log, so I fixed it.